exercises/grep/grep.spec.js

```diff
@@ -0,0 +1,233 @@
+const { spawnSync } = require('child_process')
+const path = require('path')
+
+const BASE_DIR = './tmp_exercises';
```

Comment:
This actually doesn't allow you to run this _locally_, as locally the `tmp_exercises` directory won't exist. This should probably something along the lines of:

```javascript
const BASE_DIR = path.resolve(__dirname)
```

... this way it doesn't matter _where_ the `cwd()` is, because `__dirname` will be the `path.dirname` of the _current file_!

Reply:
Indeed, this is much cleaner, thanks :)